Polyhedron/demo/Polyhedron/Plugins/Point_set/Point_set_normal_estimation_plugin.ui

```diff
@@ -46,7 +46,7 @@
       <item row="0" column="1">
        <widget class="QSpinBox" name="m_pca_neighbors">
         <property name="suffix">
-         <string>Nearest Neighbors</string>
+         <string> Nearest Neighbors</string>
```

Comment:
I am surprised by this modification. Was that on purpose? If it is, what is the goal of inserting one blank space of unknown width?